css-flexbox/align-items.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      min-height: 100vh;
+
+    }
+    .card-container{
+      flex-basis: calc(100% / 3);
+      align-items: center;
```

Comment:
The `align-items` and `justify-content` properties must be set on the flex container itself, not its children. These should be set on the `#card-deck` rule. Also, please provide a picture of your output for this file

Reply:
![Screen Shot 2021-01-23 at 3 43 49 PM](https://user-images.githubusercontent.com/69738204/105613714-548bf980-5d92-11eb-88fb-638b2b238feb.png)